data_visualization_in_open_source_software/data_visualization.md

```diff
@@ -0,0 +1,265 @@
+
+- Informative figure captions, alt text (for online visuals), or note text explaining the visualization
+- A "headline" title for your visualization that articulates the take-home message
+- A data table presenting the data shown in the chart (note that this may be in an appendix or similar)
+- A summary in your text of the important information in the visualization
+
+For more guidance on accessible design of data visualizations, see this presentation by [Kent Eisenhuth](http://kenteisenhuth.com/), a user experience designer at Google. It's set to start at his discussion of how to provide a really valuable summary of a data visualization, but if you're interested you may like to scroll back and watch it from the beginning.
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/4zV9XqK0foM?start=2940" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+## Quiz
+
+Which of the following statements best describes the seaborn and ggplot2 libraries?
+
+[(X)] They are designed to integrate well with other popular data science libraries
+[( )] They have basic plotting functions that are quick to use but offer little control
+[( )] They come standard in base python or R, respectively, so you don't have to install them before use
+***********************************************************************
+<div class = "answer">
+Both seaborn and ggplot2 are powerful libraries designed for use in data science. Because they offer so much flexibility, they can be difficult to learn initially, but if you'll be producing data visualizations for use in publications and presentations you'll be glad you put in the time to learn to use them.
+
+Neither ggplot2 nor seaborn are included when you install R or python, so you'll need to install them before the first time you use them. You should also periodically check for updates and install the newest versions of these and other packages you rely on.
+</div>
+***********************************************************************
+
+True or False: It's generally NOT necessary to reformat data before plotting.
+
+[( )] TRUE
+[(X)] FALSE
+***********************************************************************
+<div class = "answer">
+Although some data reformatting can be completed as part of the plotting code, in many cases it will be necessary (or at least a lot easier!) to do any data manipulation before plotting.
+</div>
+***********************************************************************
+
+List as many plot elements as you can that can be used as tools to communicate data values.
```

Comment:
in the learning objectives and here they're referred to as plot elements, but in the content they're referred to exclusively as tools -- we may want to have a constant approach (personally my vote is for element > tool)